aaelf64/aaelf64.rst

```diff
@@ -940,6 +940,13 @@ A ``RELA`` format relocation must be used if the initial addend cannot be encode
 
 There is no PC bias to accommodate in the relocation of a place containing an instruction that formulates a PC- relative address. The program counter reflects the address of the currently executing instruction.
 
+There are two special cases for forming the initial addend of REL-type relocations where the immediate field cannot normally hold small signed integers:
+
+* For relocations processing MOVZ and MOVK instructions, the initial   addend is formed by interpreting the 16-bit literal field of the instruction as a 16-bit signed value in the range -32768 <= A < 32768. The interpretation is the same whether or not the instruction applies a left shift to its immediate: the addend is never treated as shifted.
```

Comment:
Would it be worth including MOV (wide immediate) using the Arm ARM description. For example 
```For relocations processing MOV (wide immediate), MOVZ and MOVK instructions,```

MOV (wide immediate) is an alias for MOVZ so this is included in MOVZ and MOVK, but MOV is used a lot in code examples.

Reply:
I dithered about that myself. Added a parenthesis.